images/linux/toolsets/toolset-1804.json

```diff
@@ -167,7 +167,6 @@
             "time",
             "unzip",
             "wget",
-            "yamllint",
```

Comment:
I think it makes sense to create pipx_packages list in the toolset and add yamllint and aws-sam-cli to it.
Likely, these are not the last tools we will install with pipx

Reply:
Done!